pom.xml

```diff
@@ -24,6 +24,7 @@
     	<openMRSVersion>2.0.5</openMRSVersion>
 		<uiframeworkVersion>3.9</uiframeworkVersion>
 		<appuiVersion>1.9.0-SNAPSHOT</appuiVersion>
+		<appframeworkVersion>2.9</appframeworkVersion>
```

Comment:
What are we using in the app framework?

Reply:
There are some classNotFound errors occured while extending the baseModuleSensitiveTest. So I just used that module